lib/pfconfig/namespaces/config/Pf.pm

```diff
@@ -113,6 +113,11 @@ sub build_child {
         $Config{$group}{$item} = [ split( /\s*,\s*/, $Config{$group}{$item}  // '' ) ];
     }
 
+    foreach my $val ("fencing.isolation_passthroughs") {
```

Comment:
just add "fencing.isolation_passthrougs" to the foreach loop above, they do the same thing